Tools/ardupilotwaf/chibios.py

```diff
@@ -283,7 +283,11 @@ def run(self):
         img2 = bytearray(img[offset+desc_len:])
         crc1 = to_unsigned(crc32(img1))
         crc2 = to_unsigned(crc32(img2))
-        githash = to_unsigned(int('0x' + os.environ.get('GIT_VERSION', self.generator.bld.git_head_hash(short=True)),16))
+        if 'GIT_VERSION' in os.environ:
+            hex_hash = os.environ.get('GIT_VERSION')
+        else:
+            hex_hash = self.generator.bld.git_head_hash(short=True)
+        githash = to_unsigned(int('0x' + hex_hash, 16))
```

Comment:
unrelated to your change but the `"0x"` is superfluous: `int(s, 16) == int("0x" + s, 16)`